### PR TITLE
Refactor Made by João Vitor Balzer

### PR DESCRIPTION
**Pull request by joaobalzer**
Refactor Made by João Vitor Balzer to the discipline of "ASPECTOS ÁGEIS DE PROGRAMAÇÃO"

### CODE REVIEW

File: Geometry/ConvexHullGraham.js [hunk 1]
```diff
@@ -24,47 +31,78 @@ function orientation (a, b, c) {
...
+        upperLenght >= 2 &&
+        orientation(
+          upperPoints[upperLenght - 2],
+          upperPoints[upperLenght - 1],
+          points[i]
+        ) === -1
+      ) {
         upperPoints.pop()
-        upLen = upperPoints.length
+        upperLenght = upperPoints.length
```
Comment (raklaptudirm) [inline Geometry/ConvexHullGraham.js:73]:
```suggestion
        upperLength = upperPoints.length
```

File: Geometry/ConvexHullGraham.js [hunk 2]
```diff
@@ -24,47 +31,78 @@ function orientation (a, b, c) {
...
 
-  for (let i = 1; i < pointsLen; i++) {
-    if (i === pointsLen - 1 || orientation(p1, points[i], p2) !== -1) {
-      let upLen = upperPoints.length
+  for (let i = 1; i < pointsLenght; i++) {
+    if (
+      i === pointsLenght - 1 ||
+      orientation(coordinateP1, points[i], coordinateP2) !== -1
+    ) {
+      let upperLenght = upperPoints.length
```
Comment (raklaptudirm) [inline Geometry/ConvexHullGraham.js:62]:
```suggestion
      let upperLength = upperPoints.length
```

File: Geometry/ConvexHullGraham.js [hunk 3]
```diff
@@ -24,47 +31,78 @@ function orientation (a, b, c) {
...
 
-      while (upLen >= 2 && orientation(upperPoints[upLen - 2], upperPoints[upLen - 1], points[i]) === -1) {
+      while (
+        upperLenght >= 2 &&
```
Comment (raklaptudirm) [inline Geometry/ConvexHullGraham.js:65]:
```suggestion
        upperLength >= 2 &&
```

**Review by raklaptudirm — CHANGES_REQUESTED** [review]

**Comment by raklaptudirm** [comment]
@joaobalzer Please rectify the changes requested.

**Comment by Bhavyajain21** [comment]
I would like to work on this issue. What's the next step I can do to  interact with you?

**Comment by raklaptudirm** [comment]
@Bhavyajain21 Which issue can you specify?

**Pull request closed without merging**